api/src/main/java/com/epam/pipeline/manager/user/UserManager.java

```diff
@@ -400,6 +428,41 @@ public boolean needToUpdateUser(List<String> groups, Map<String, String> attribu
         return new UserExporter().exportUsers(attr, users, sensitiveKeys).getBytes(Charset.defaultCharset());
     }
 
+    private PipelineUser createUser(final String name, final List<Long> roles, final List<String> groups,
+                                    final Map<String, String> attributes) {
+        Assert.isTrue(StringUtils.isNotBlank(name),
+                      messageHelper.getMessage(MessageConstants.ERROR_USER_NAME_REQUIRED));
+        String userName = name.trim().toUpperCase();
+        PipelineUser loadedUser = userDao.loadUserByName(userName);
+        Assert.isNull(loadedUser, messageHelper.getMessage(MessageConstants.ERROR_USER_NAME_EXISTS, name));
+        PipelineUser user = new PipelineUser(userName);
+        List<Long> userRoles = getNewUserRoles(roles);
+        user.setRoles(roleDao.loadRolesList(userRoles));
+        user.setGroups(groups);
+        user.setAttributes(attributes);
+        storageValidator.validate(user);
```

Comment:
Validating here is useless since `defaultStorageId` is handled only in `createUserDefaultStorage ` method

Reply:
Done